straxen/url_config.py

```diff
@@ -264,14 +269,16 @@ def evaluate_dry(cls, url: str, **kwargs):
         :keyword: any additional kwargs are passed to self.dispatch (see example)
         :return: evaluated value of the URL.
         """
-        if cls.PLUGIN_ATTR_PREFIX in url:
-            raise ValueError(
-                'You specified at least one parameter that depends on the '
-                'plugin. We cannot fetch this in the dry run. Try replacing '
-                'e.g. plugin.run_id=027000 with run_id=027000 (or pass as an '
-                'extra kwargs)')
         url_arg, url_kwarg = cls.split_url_kwargs(url)
-        return cls.dispatch(cls, url_arg, **url_kwarg, **kwargs)
+
+        combined_kwargs = dict(url_kwarg, **kwargs)
```

Comment:
Do you mean?
` combined_kwargs = dict(**url_kwarg, **kwargs)`

Reply:
ah you are right, I'm confused with {**kwargs, **more_kwargs}
![image](https://user-images.githubusercontent.com/22295914/192796494-13180f73-74dc-4fc6-87ca-44078c15e160.png)

this is so much easier in python 3.10
![image](https://user-images.githubusercontent.com/22295914/192796565-bd4aec90-8c33-49b4-8d77-e2ab2ddab188.png)

Reply:
I only know this because i tried overriding a value and got an error :sweat_smile: